account_analytic_business_unit/models/account_move_line.py

```diff
@@ -5,6 +5,9 @@ class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
     analytic_line_count = fields.Integer(compute="_compute_total_analytic_lines")
+    business_unit_id = fields.Many2one(
+        related="analytic_account_id.business_unit_id", store=True, inverse="_inverse_analytic_account_id"
```

Comment:
why the invserse?

Reply:
pls add a help string

Reply:
if you added it to allow edit it  then  ensure the record related don't change when modifying this